deploy/eck-elasticsearch/values.yaml

```diff
@@ -32,31 +32,79 @@ annotations: {}
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-stack-monitoring.html
 #
 monitoring: {}
+  # metrics:
+  #   elasticsearchRefs:
+  #   - name: monitoring
+  #     namespace: observability 
+  # logs:
+  #   elasticsearchRefs:
+  #   - name: monitoring
+  #     namespace: observability
 
 # Control the Elasticsearch transport module used for internal communication between nodes.
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-transport-settings.html
 #
 transport: {}
+  # service:
+  #   metadata:
+  #     labels:
+  #       my-custom: label
+  #   spec:
+  #     type: LoadBalancer
+  # tls:
+  #   subjectAltNames:
+  #     - ip: 1.2.3.4
+  #     - dns: hulk.example.com
+  #   certificate:
+  #     secretName: custom-ca
 
 # Settings to control how Elasticsearch will be accessed.
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-accessing-elastic-services.html
 #
 http: {}
+  # service:
+  #   metadata:
+  #     labels:
+  #       my-custom: label
+  #   spec:
+  #     type: LoadBalancer
+  # tls:
+  #   subjectAltNames:
+  #     - ip: 1.2.3.4
+  #     - dns: hulk.example.com
+  #   certificate:
+  #     secretName: custom-ca
 
 # Control Elasticsearch Secure Settings.
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-es-secure-settings.html#k8s-es-secure-settings
 #
 secureSettings: {}
+  # - secretName: one-secure-settings-secret
+  # Projection of secret keys to specific paths
+  # - secretName: gcs-secure-settings
+  #   entries:
+  #   - key: gcs.client.default.credentials_file
+  #   - key: gcs_client_1
+  #     path: gcs.client.client_1.credentials_file
+  #   - key: gcs_client_2
+  #     path: gcs.client.client_2.credentials_file
 
 # Settings for limiting the number of simultaneous changes to an Elasticsearch resource.
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-update-strategy.html
 #
 updateStrategy: {}
+  # changeBudget:
+  #     maxSurge: 3
+  #     maxUnavailable: 1
```

Comment:
Indentation feels off here.

Reply:
If I select the commented out sections, and use my editor to remove all comments, it becomes perfectly indented.  My intent here was to make it easy to remove the comments using some editor, and have them aligned correctly.

Were you wanting to see them more like the following?
```
updateStrategy: {}
# changeBudget:
#     maxSurge: 3
#     maxUnavailable: 1
```

Reply:
Sorry maybe it is just the diff UI here in GH as it looks like it is indented by 4 spaces instead of 2.

Reply:
@thbkrkr pointed out what I was not seeing.  I thought y'all were referring to the comment's indentation, not the `maxSurge, maxUnavailable` fields.  I've fixed this.